libs/auth/scripts/src/check_pin.ts

```diff
@@ -21,12 +21,11 @@ function parseCommandLineArgs(args: readonly string[]): CheckPinInput {
 }
 
 async function checkPin({ cardType }: CheckPinInput): Promise<void> {
+  const rl = readline.createInterface(process.stdin, process.stdout);
   const pin = await new Promise<string>((resolve) => {
-    createInterface(process.stdin, process.stdout).question(
-      'Enter PIN: ',
-      resolve
-    );
+    rl.question('Enter PIN: ', resolve);
   });
+  rl.close();
```

Comment:
Unrelated, just adding a missing readline interface close. Isn't causing a problem right now but would if we ever added another prompt to this script